doc/fd.1

```diff
@@ -0,0 +1,106 @@
+.TH FD 1 fd-4.0.0
+.SH NAME
+fd \- find files
```

Comment:
Maybe this could be a little bit more detailed: "fd - find entries in the filesystem"?

Reply:
I took that from the posix man page and sbase man page. Both uses that.